modules/disconnected-osus-overview.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding-openshift-updates.adoc
+
+:_content-type: CONCEPT
+[id="update-service-overview_{context}"]
+
+= Using the OpenShift Update Service in a disconnected environment
```

Comment:
should OpenShift here be {product-title}? and there are other mentions of OpenShift in here too should those be {product-title}?

Reply:
I am not aware of one, at least not that I can see in the attributes file. You could always create an attribute for OSUS if you really wanted to (it would alleviate room for typos and such, but not necessary). If OpenShift Container Platform Update Service is incorrect you can disregard my comment.